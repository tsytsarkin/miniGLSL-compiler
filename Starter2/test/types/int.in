{
    int i;
    int j = 123;
    j = 93;
}
