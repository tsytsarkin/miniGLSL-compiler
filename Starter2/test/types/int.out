  1: {
  2:     int i;
  3:     int j = 123;
  4:     j = 93;
  5: }
empty declarations
INT_T type
declaration
declarations
INT_T type
INT_C expression
assignment declaration
declarations
empty statements
ID variable
INT_C expression
assignment statement
statements
scope
program
